
---

sick_safetyscanners/FieldMsg[] fields
string device_name
sick_safetyscanners/MonitoringCaseMsg[] monitoring_cases
